components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -231,6 +231,15 @@ class BrowserToolbar @JvmOverloads constructor(
             editToolbar.urlView.setTextColor(value)
         }
 
+    /**
+     * Sets the colour of the tracking protection icon.
+     */
+    var trackingProtectionColor: Int = displayToolbar.defaultColor
+        set(value) {
```

Comment:
I think you meant to put the default value on the DisplayToolbar like the other APIs and add a `get()` here?

Reply:
Done!